WalletConnect/Models/Trust/WCTrustAccount.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  WCTrustAccount.swift
+//  WalletConnect
+//
+//  Created by Leone Parise on 25/06/19.
+//
+
+import Foundation
+
+public struct WCTrustAccount: Codable {
+    public let network: UInt32
```

Comment:
can we do `UInt64` for consistency

Reply:
But `CoinType` raw value is `UInt32`. I used `UInt32` to avoid unnecessary casts.